bazel/grpc_deps.bzl

```diff
@@ -185,6 +185,13 @@ def grpc_deps():
             sha256 = "ee854b5de299138c1f4a2edb5573d22b21d975acfc7aa938f36d30b49ef97498",
         )
 
+    if "bazel_skylib" not in native.existing_rules():
+        git_repository(
```

Comment:
it looks like everything else is a added via an http_archive  (downloaded from github). Let's change to that?

Reply:
Done